Os passos para meter a app iOS a correr não funcionam

1. `npm install` dá erro, é preciso correr `nem install --force`:
```
npm error code ERESOLVE
npm error ERESOLVE could not resolve
npm error
npm error While resolving: gira@1.0.3
npm error Found: @sveltejs/adapter-auto@2.1.1
npm error node_modules/@sveltejs/adapter-auto
npm error   dev @sveltejs/adapter-auto@"^3.0.0" from the root project
npm error
npm error Could not resolve dependency:
npm error dev @sveltejs/adapter-auto@"^3.0.0" from the root project
npm error
npm error Conflicting peer dependency: @sveltejs/kit@2.6.1
npm error node_modules/@sveltejs/kit
npm error   peer @sveltejs/kit@"^2.0.0" from @sveltejs/adapter-auto@3.2.5
npm error   node_modules/@sveltejs/adapter-auto
npm error     dev @sveltejs/adapter-auto@"^3.0.0" from the root project
```

2. `npx cap run ios` não funciona se estiver a correr na pasta `ios/App`, e quando se corre no raiz do projeto corre, mas depois quando é selecionado um simulador:
```
✔ Please choose a target device: › iPhone 15 (simulator) (A0F10052-2700-4E2B-ADE6-D9304E5D6397)
✖ Running xcodebuild - failed!
[error] Command line invocation:
        /Applications/Xcode-15.4.0.app/Contents/Developer/usr/bin/xcodebuild -workspace App.xcworkspace -scheme App
        -configuration Debug -destination id=A0F10052-2700-4E2B-ADE6-D9304E5D6397 -derivedDataPath
        /Users/michael/Documents/Projects/gira/ios/DerivedData/A0F10052-2700-4E2B-ADE6-D9304E5D6397

        User defaults from command line:
        IDEDerivedDataPathOverride =
        /Users/michael/Documents/Projects/gira/ios/DerivedData/A0F10052-2700-4E2B-ADE6-D9304E5D6397
        IDEPackageSupportUseBuiltinSCM = YES

        2024-09-30 16:18:49.254 xcodebuild[1265:37106696] Writing error result bundle to
        /var/folders/90/sfr5wzqs5ws520tlls7m15300000gn/T/ResultBundle_2024-30-09_16-18-0049.xcresult
        xcodebuild: error: The workspace named "App" does not contain a scheme named "App". The "-list" option can be
        used to find the names of the schemes in the workspace.
```

Seria possível as instruções serem corrigidas?

--- a/README.md
+++ b/README.md
@@ -78,16 +78,12 @@ cd gira
 ```sh
 npm install
 npm run build
-cd ios/App
-pod install
 npx cap run ios
 ```
   ou
 ```bash
 bun install
 bun run build
-cd ios/App
-pod install
 bunx cap run ios
 ```
 

--- a/capacitor.config.json
+++ b/capacitor.config.json
@@ -10,8 +10,11 @@
 		"Keyboard": {
 			"resize": "none"
 		}
-  },
-  "android": {
-    "useLegacyBridge": true
+	},
+	"android": {
+		"useLegacyBridge": true
+	},
+	"ios": {
+		"scheme": "GiraPlus"
 	}
 }

--- a/ios/App/App.xcodeproj/project.pbxproj
+++ b/ios/App/App.xcodeproj/project.pbxproj
@@ -14,11 +14,13 @@
 		504EC30F1FED79650016851F /* Assets.xcassets in Resources */ = {isa = PBXBuildFile; fileRef = 504EC30E1FED79650016851F /* Assets.xcassets */; };
 		504EC3121FED79650016851F /* LaunchScreen.storyboard in Resources */ = {isa = PBXBuildFile; fileRef = 504EC3101FED79650016851F /* LaunchScreen.storyboard */; };
 		50B271D11FEDC1A000F3C39B /* public in Resources */ = {isa = PBXBuildFile; fileRef = 50B271D01FEDC1A000F3C39B /* public */; };
-		A084ECDBA7D38E1E42DFC39D /* Pods_App.framework in Frameworks */ = {isa = PBXBuildFile; fileRef = AF277DCFFFF123FFC6DF26C7 /* Pods_App.framework */; };
+		B476283CDBC8EF936FF4DB07 /* Pods_GiraPlus.framework in Frameworks */ = {isa = PBXBuildFile; fileRef = 3C875F7D51FBE5222392B3C4 /* Pods_GiraPlus.framework */; };
 /* End PBXBuildFile section */
 
 /* Begin PBXFileReference section */
+		0C7AA70149B8DB8601745EF4 /* Pods-GiraPlus.release.xcconfig */ = {isa = PBXFileReference; includeInIndex = 1; lastKnownFileType = text.xcconfig; name = "Pods-GiraPlus.release.xcconfig"; path = "Pods/Target Support Files/Pods-GiraPlus/Pods-GiraPlus.release.xcconfig"; sourceTree = "<group>"; };
 		2FAD9762203C412B000D30F8 /* config.xml */ = {isa = PBXFileReference; lastKnownFileType = text.xml; path = config.xml; sourceTree = "<group>"; };
+		3C875F7D51FBE5222392B3C4 /* Pods_GiraPlus.framework */ = {isa = PBXFileReference; explicitFileType = wrapper.framework; includeInIndex = 0; path = Pods_GiraPlus.framework; sourceTree = BUILT_PRODUCTS_DIR; };
 		50379B222058CBB4000EE86E /* capacitor.config.json */ = {isa = PBXFileReference; fileEncoding = 4; lastKnownFileType = text.json; path = capacitor.config.json; sourceTree = "<group>"; };
 		504EC3041FED79650016851F /* GiraPlus.app */ = {isa = PBXFileReference; explicitFileType = wrapper.application; includeInIndex = 0; path = GiraPlus.app; sourceTree = BUILT_PRODUCTS_DIR; };
 		504EC3071FED79650016851F /* AppDelegate.swift */ = {isa = PBXFileReference; lastKnownFileType = sourcecode.swift; path = AppDelegate.swift; sourceTree = "<group>"; };
@@ -27,8 +29,8 @@
 		504EC3111FED79650016851F /* Base */ = {isa = PBXFileReference; lastKnownFileType = file.storyboard; name = Base; path = Base.lproj/LaunchScreen.storyboard; sourceTree = "<group>"; };
 		504EC3131FED79650016851F /* Info.plist */ = {isa = PBXFileReference; lastKnownFileType = text.plist.xml; path = Info.plist; sourceTree = "<group>"; };
 		50B271D01FEDC1A000F3C39B /* public */ = {isa = PBXFileReference; lastKnownFileType = folder; path = public; sourceTree = "<group>"; };
-		AF277DCFFFF123FFC6DF26C7 /* Pods_App.framework */ = {isa = PBXFileReference; explicitFileType = wrapper.framework; includeInIndex = 0; path = Pods_App.framework; sourceTree = BUILT_PRODUCTS_DIR; };
 		AF51FD2D460BCFE21FA515B2 /* Pods-App.release.xcconfig */ = {isa = PBXFileReference; includeInIndex = 1; lastKnownFileType = text.xcconfig; name = "Pods-App.release.xcconfig"; path = "Pods/Target Support Files/Pods-App/Pods-App.release.xcconfig"; sourceTree = "<group>"; };
+		C86272E4D774FD869AF43BBF /* Pods-GiraPlus.debug.xcconfig */ = {isa = PBXFileReference; includeInIndex = 1; lastKnownFileType = text.xcconfig; name = "Pods-GiraPlus.debug.xcconfig"; path = "Pods/Target Support Files/Pods-GiraPlus/Pods-GiraPlus.debug.xcconfig"; sourceTree = "<group>"; };
 		FC68EB0AF532CFC21C3344DD /* Pods-App.debug.xcconfig */ = {isa = PBXFileReference; includeInIndex = 1; lastKnownFileType = text.xcconfig; name = "Pods-App.debug.xcconfig"; path = "Pods/Target Support Files/Pods-App/Pods-App.debug.xcconfig"; sourceTree = "<group>"; };
 /* End PBXFileReference section */
 
@@ -37,7 +39,7 @@
 			isa = PBXFrameworksBuildPhase;
 			buildActionMask = 2147483647;
 			files = (
-				A084ECDBA7D38E1E42DFC39D /* Pods_App.framework in Frameworks */,
+				B476283CDBC8EF936FF4DB07 /* Pods_GiraPlus.framework in Frameworks */,
 			);
 			runOnlyForDeploymentPostprocessing = 0;
 		};
@@ -47,7 +49,7 @@
 		27E2DDA53C4D2A4D1A88CE4A /* Frameworks */ = {
 			isa = PBXGroup;
 			children = (
-				AF277DCFFFF123FFC6DF26C7 /* Pods_App.framework */,
+				3C875F7D51FBE5222392B3C4 /* Pods_GiraPlus.framework */,
 			);
 			name = Frameworks;
 			sourceTree = "<group>";
@@ -90,6 +92,8 @@
 			children = (
 				FC68EB0AF532CFC21C3344DD /* Pods-App.debug.xcconfig */,
 				AF51FD2D460BCFE21FA515B2 /* Pods-App.release.xcconfig */,
+				C86272E4D774FD869AF43BBF /* Pods-GiraPlus.debug.xcconfig */,
+				0C7AA70149B8DB8601745EF4 /* Pods-GiraPlus.release.xcconfig */,
 			);
 			name = Pods;
 			sourceTree = "<group>";
@@ -178,7 +182,7 @@
 			);
 			name = "[CP] Check Pods Manifest.lock";
 			outputPaths = (
-				"$(DERIVED_FILE_DIR)/Pods-App-checkManifestLockResult.txt",
+				"$(DERIVED_FILE_DIR)/Pods-GiraPlus-checkManifestLockResult.txt",
 			);
 			runOnlyForDeploymentPostprocessing = 0;
 			shellPath = /bin/sh;
@@ -197,7 +201,7 @@
 			);
 			runOnlyForDeploymentPostprocessing = 0;
 			shellPath = /bin/sh;
-			shellScript = "\"${PODS_ROOT}/Target Support Files/Pods-App/Pods-App-frameworks.sh\"\n";
+			shellScript = "\"${PODS_ROOT}/Target Support Files/Pods-GiraPlus/Pods-GiraPlus-frameworks.sh\"\n";
 			showEnvVarsInLog = 0;
 		};
 /* End PBXShellScriptBuildPhase section */
@@ -342,7 +346,7 @@
 		};
 		504EC3171FED79650016851F /* Debug */ = {
 			isa = XCBuildConfiguration;
-			baseConfigurationReference = FC68EB0AF532CFC21C3344DD /* Pods-App.debug.xcconfig */;
+			baseConfigurationReference = C86272E4D774FD869AF43BBF /* Pods-GiraPlus.debug.xcconfig */;
 			buildSettings = {
 				ASSETCATALOG_COMPILER_APPICON_NAME = AppIcon;
 				CODE_SIGN_STYLE = Automatic;
@@ -367,7 +371,7 @@
 		};
 		504EC3181FED79650016851F /* Release */ = {
 			isa = XCBuildConfiguration;
-			baseConfigurationReference = AF51FD2D460BCFE21FA515B2 /* Pods-App.release.xcconfig */;
+			baseConfigurationReference = 0C7AA70149B8DB8601745EF4 /* Pods-GiraPlus.release.xcconfig */;
 			buildSettings = {
 				ASSETCATALOG_COMPILER_APPICON_NAME = AppIcon;
 				CODE_SIGN_STYLE = Automatic;

